common/config/mdl.rb

```diff
@@ -1,7 +1,6 @@
 all
 rule 'MD002', :level => 1
 rule 'MD007', :indent => 4
-rule 'MD009', :br_spaces => 2
```

Comment:
Couldn't you add this line to istio/common-files instead of removing this?

Reply:
You are removing the line, not adding it. Normally we don't remove checks to allow more things pass.

Reply:
Ok, current formatting looks good as well.